x/slashing/keeper/signing_info.go

```diff
@@ -83,6 +83,17 @@ func (k Keeper) SetMissedBlockBitmapChunk(ctx context.Context, addr sdk.ConsAddr
 // IndexOffset modulo SignedBlocksWindow. This index is used to fetch the chunk
 // in the bitmap and the relative bit in that chunk.
 func (k Keeper) GetMissedBlockBitmapValue(ctx context.Context, addr sdk.ConsAddress, index int64) (bool, error) {
+	// check the key rotated, if rotated use the returned consKey to get the missed blocks
+	// because missed blocks are still pointing to the old key
+	oldPk, err := k.sk.GetNewToOldConsKeyMap(ctx, addr)
+	if err != nil {
```

Comment:
do we need to check for errnotfound here?

Reply:
It's being check at `GetNewToOldConsKeyMap`, no error will be return in case of ErrNotFound, just a `nil, nil`, lgtm